src/internal/react-enhancer.ts

```diff
@@ -57,15 +57,23 @@ const reactEnhancer = <T>(
   }) as BehaviorObservable<T>
 
   let promise: any
+  let hasError = false
+  let error: unknown = undefined
   const getValue = () => {
+    if (hasError) {
+      throw error
+    }
     try {
       return (source$ as BehaviorObservable<T>).getValue()
     } catch (e) {
       if (promise) throw promise
 
       if (!IS_SSR && e !== SUSPENSE) {
         source$
-          .pipe(takeUntil(race(onSubscribe, of(true).pipe(delay(60000)))))
+          .pipe(
+            takeUntil(race(onSubscribe, of(true).pipe(delay(60000)))),
+            catchError(() => of(true)),
+          )
           .subscribe()
```

Comment:
This was needed because otherwise this subscription would receive an error without handling it - Raising an uncaught exception that jest would pick up as `error` even if (`expect(true).toBe(true)`).